### PR TITLE
Feat/new topic for Kinde MFA

### PR DESCRIPTION
**Pull request by clairekinde11**
New topic explaining how business owners can request MFA for accessing their Kinde business

## Summary by CodeRabbit

- **New Features**
	- Introduced a new documentation page on implementing multi-factor authentication (MFA) for Kinde access.
	- Outlined the process for teams to request MFA, including options for mandatory or role-specific application.
	- Provided guidance on necessary information for MFA requests, enhancing security protocols for users.

### CODE REVIEW

File: src/content/docs/get-started/team-and-account/mfa-for-kinde-access.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+We can set MFA so that it applies to everyone in your team, or only applies to certain roles, e.g. Owners within the business. 
+
+## What to ask for when you request MFA
+
+Send your MFA request to support@kinde.com, and include:
+
+- The name of each business that MFA should apply to
+- The MFA factors that you want your team to have available to them, e.g. email code, SMA code, Authenticator app.
+- Whether you want MFA to be opt-in or required
```
Comment (oliwolff1) [inline src/content/docs/get-started/team-and-account/mfa-for-kinde-access.mdx:19]: @clairekinde11 - we cant allow `optional` mfa for this. I would exclude this line

File: src/content/docs/get-started/team-and-account/mfa-for-kinde-access.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+If you want your team members to use multi-factor authentication (MFA) when they sign in to Kinde, you can contact us to request this. 
+
+We can set MFA so that it applies to everyone in your team, or only applies to certain roles, e.g. Owners within the business. 
+
+## What to ask for when you request MFA
+
+Send your MFA request to support@kinde.com, and include:
+
+- The name of each business that MFA should apply to
+- The MFA factors that you want your team to have available to them, e.g. email code, SMA code, Authenticator app.
```
Comment (oliwolff1) [inline src/content/docs/get-started/team-and-account/mfa-for-kinde-access.mdx:18]: Should we indicate here that you could have 1 option, or offer 2 or even all 3?

**Review by oliwolff1 — COMMENTED** [review]
Few minor comments

**Comment by clairekinde11** [comment]
Oli feedback completed.

**Review by marcosmartini — APPROVED** [review]
LGTM - @oliwolff1 anything else to add?

**Pull request merged**